tests/database/serverConfig.test.ts

```
@@ -0,0 +1,182 @@
+import Database from '../../database/Database';
+import { Database as BunDatabase } from 'bun:sqlite';
```

ESLint

[error] `bun:sqlite` import should occur before import of `../../database/Database`

(import/order)